src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
I am not sure I understand why the behavior here is different for WIN64EXCEPTIONS. This UpdateRegDisplay is essentially a simple unwind to the caller context. So I would expect it to behave as the UnwindStackFrame w.r.t. the semantics of the value it sets to the SP.